src/components/Categories.js

```diff
@@ -1,7 +1,22 @@
-const Categories = () => (
-  <>
-    <p>Here goes the Categories</p>
-  </>
-);
+import React from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { updateStatus } from '../redux/categories/categoriesSlice';
+
+const Categories = () => {
+  const { message } = useSelector((store) => store.categories);
+  const dispatch = useDispatch();
+  const handleClick = () => {
+    dispatch(updateStatus());
+  };
+
+  return (
+    <div>
+      <button onClick={handleClick} type="button">
+        More Items
+      </button>
+      <h2>{message}</h2>
+    </div>
+  );
+};
```

Comment:
- [ ] **[OPTIONAL]** Could you please take a quick look at your categories page? I seem to be getting redirected to an error.

   ![image](https://github.com/firstpj/Bookstore/assets/107952724/bf7d29b4-f08e-4a41-80aa-f2a149c49119)